### ISSUE TITLE
Abnormal circulating Human Epidermal Growth Factor Receptor 2 concentration

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#5998]
**Preferred term label:** Abnormal circulating Human Epidermal Growth Factor Receptor 2 concentration

**Synonyms** Abnormal circulating HER2

**Definition (free text, please give PubMed ID)** Concentration of human epidermal growth factor receptor 2 in the blood is outside the normal range., PMID: 12181714

**Parent term (use hpo.jax.org/app)** Abnormal circulating protein level HP:0010876

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by JillMiller** [comment]
**Preferred term label:** Increased circulating Human Epidermal Growth Factor Receptor 2 concentration

**Synonyms** Increased circulating HER4

**Definition (free text, please give PubMed ID)** Retrieving data. Wait a few seconds and try to cut or copy again.

**Parent term (use hpo.jax.org/app)** Abnormal circulating Human Epidermal Growth Factor Receptor 2 concentration

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@JillMiller  I do not thing that EGFR2 is measured in the circulation clinically. Is this somehow being measured in the NBSTRN? Can you provide more context?

**Comment by pnrobinson** [comment]
Not a clinical test, closing
